images/s6_assets/test.sh

```diff
@@ -69,10 +69,12 @@ if [[ ${image} != *"galaxy"* ]];then
   if [ -d pulp-cli ]; then
```

Comment:
It sounds strange that we do not even know whether we have the cli installed at this time.
Can we just do the same as we do in the plugin_ci and use a released version of the CLI?

Reply:
I think it is because this script is ran multiple times during the CI, thus the need to check.